src/main/com/wsscode/misc/coll.cljc

```diff
@@ -1,7 +1,11 @@
 (ns com.wsscode.misc.coll
   (:require
     [clojure.set :as set])
-  #?(:clj
+  #?(:bb
```

Comment:
@wilkerlucio Why is this needed? Same as `:clj` ?

Reply:
Its not, if you expand the CLJ version, it imports more classes, that fail to import in Babashka

Reply:
The `clojure.lang.RT` and `java.util.Iterator` classes